loadtest/helper.go

```diff
@@ -0,0 +1,39 @@
+package loadtest
+
+import (
+	"math/rand"
+
+	uuid "github.com/satori/go.uuid"
+)
+
+func getRandomScore() int {
+	return rand.Intn(1000)
+}
+
+func getRandomPlayerName() string {
+	return "PlayerName"
```

Comment:
"PlayerName" is not actually random. Why not "PlayerName-XYZ" where XYZ can be the first substring of an uuid.

Reply:
Sure!